LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 Anton Korzunov
```

Comment:
Technically, `2017-2023` :D

Reply:
Then one needs a bot updating it every year 😞